artifactory/utils/buildutils_test.go

```diff
@@ -0,0 +1,114 @@
+package utils
+
+import (
+	"os"
+	"path/filepath"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	artclientutils "github.com/jfrog/jfrog-client-go/artifactory/services/utils"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var timestamp = strconv.FormatInt(time.Now().Unix(), 10)
+
+func TestGetBuildName(t *testing.T) {
+	const buildName = "buildName1"
+	const buildNameEnv = "envBuildName"
+	const buildNameFile = "fileBuildName"
+
+	// Setup global build name env var.
+	// Ensure other parallel tests wont get effected.
```

Comment:
Ensure other parallel tests wont get effected.
-->
Ensure that other parallel tests won't be effected.